src/components/config-editor/AdditionalSettings.tsx

```diff
@@ -14,26 +14,41 @@ interface AdditionalSettingsProps {
 }
 
 export function AdditionalSettings({ jsonData, onOptionChange }: AdditionalSettingsProps) {
-  return config.secureSocksDSProxyEnabled && isVersionGtOrEq(config.buildInfo.version, '10.0.0') ? (
+  const shouldShowSection = config.secureSocksDSProxyEnabled && isVersionGtOrEq(config.buildInfo.version, '10.0.0');
```

Comment:
I believe this should always be shown now that the TLS skipping option has been added.